src/contract.rs

```diff
@@ -14,7 +10,7 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
     _env: Env,
     info: MessageInfo,
     msg: InitMsg,
-) -> StdResult<InitResponse> {
+) -> InitResult {
```

Comment:
Please look at the issue again.

There was some discussion yesterday, and I think we prefer to move towards: `Result<InitResponse, StdError>` for consistency between all types.